### DISCUSSION TITLE
Opening issue for web based docs?

### DISCUSSION THREAD

**Discussion by jmaslanko** [pydata/xarray#9245] [General] [answered]
Hello 👋  I am a new xarray user and am hoping to make my first open source contribution!  However, the issue I found is in the style sheets for docs and was not sure how to approach creating the issue?  When you view the [docs](https://docs.xarray.dev/), the secondary sidebar on the right side of the screen does not collapse on mobile or a narrow browser.  Would love to help fix the issue, if someone could help point me in the direction of contributing I would appreciate it since its not an issue with the core xarray package.  Thanks!

**Comment by TomNicholas** [comment, marked as answer]
Hi @jmaslanko! Thanks for your interest in contributing :)

Those docs are actually part of this repository, in the docs directory. To find out how to make changes to the documentation, have a look at our contributing guide, which is another page in the docs.
